rmgpy/tools/canteramodel.py

```diff
@@ -363,7 +367,13 @@ def plot(self, data, top_species=10, top_sensitive_reactions=10):
                 ReactionSensitivityPlot(x_var=time,
                                         y_var=reaction_sensitivity_data[j * num_ct_reactions:(j + 1) * num_ct_reactions],
                                         num_reactions=top_sensitive_reactions).barplot(
-                    os.path.join(self.output_directory, '{0}_{1}_sensitivity.png'.format(i + 1, species.to_chemkin())))
+                    os.path.join(self.output_directory, '{0}_{1}_reaction_sensitivity.png'.format(i + 1, species.to_chemkin())))
+                if self.thermo_SA:
+                    ThermoSensitivityPlot(x_var=time,
+                                          y_var=thermodynamic_sensitivity_data[j * num_ct_species:(j + 1) * num_ct_species],
+                                          xlabel='dln(c)/d(H_i) [(J/mol)^-1]',
```

Comment:
I think kcal/mol is much preferable to J/mol for most kineticists, it has better known benchmarks, definitely should at least use kJ/mol.

Reply:
Thanks for sharing. I updated the units to plot in [(kcal/mol)^-1]